app/src/main/java/ar/com/wolox/android/training/ui/training/LoginActivity.java

```diff
@@ -0,0 +1,137 @@
+package ar.com.wolox.android.training.ui.training;
+
+import android.content.ActivityNotFoundException;
+import android.content.ComponentName;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.util.Log;
+import android.util.Patterns;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.google.android.material.textfield.TextInputEditText;
+import com.google.android.material.textfield.TextInputLayout;
+
+import ar.com.wolox.android.R;
+
+/**
+ *   Login activity
+ */
+public class LoginActivity extends AppCompatActivity implements View.OnClickListener {
```

Comment:
We use to have one WolmoActivity that contains a WolmoFragment. So change it to make that. You can copy it from the Example.